starboard/shared/posix/impl/file_flush.h

```diff
@@ -36,7 +37,18 @@ bool FileFlush(SbFile file) {
     return false;
   }
 
-  return !HANDLE_EINTR(fsync(file->descriptor));
+  // Open a file stream by descriptor
+  FILE* stream = fdopen(file->descriptor, "a");
```

Comment:
I'm pretty sure that there is no data previously written to the fd for the same file that will be flushed by this, and that fflush will only flush data buffered for the "FILE*"  stream, the concept for which only lives in libc outside of the kernel.

Separately, this would be racy and therefore not reliable which could be an issue if another thread deletes or worse renames the file before we open a stream to it.

Reply:
It results in a `write(fd,0,0)` call to kernel, if done properly - this implementation doesn't do it properly. 

And if we were to use that, it's simpler to just call `write(fd,0,0)`. Thats why i closed the PR